packages/lit-dev-content/site/docs/localization/transform-mode.md

```diff
@@ -54,8 +54,8 @@ In your JavaScript or TypeScript project, optionally call
 `configureTransformLocalization`, passing an object with the following
 properties:
 
-- `sourceLocale: string`: Required locale code in which source templates in this
-  project are written.
+- `sourceLocale: string`: Locale in which source templates are written.
+  Specified as a locale code (for example: `"en"`).
 
 ```js
```

Comment:
Code sample formatting seems odd, following directly after a bullet list. It's not clear whether it's supposed to be part of the description of `sourceLocale`, or a more general example.

If it's supposed to belong to `sourceLocale`, it should be indented 4 here so it lines up under the bullet text. If not, maybe add a sentence introducing it. Or even a fragment like, "For example: "

Reply:
Also, as with the similar section in runtime mode, it's odd that the properties of the input object are shown as bullets, and the properties of the output object are shown as separate sections.  These should be consistent.

Reply:
Done, done